fluid/mnist.py

```
@@ -161,6 +169,11 @@ def run_benchmark(model, args):
             end = time.time()
             loss = np.array(outs[0])
             acc = np.array(outs[1])
+
+            loss_scalar.add_record(step, loss)
```

loss 必须是一个 scalar，就是个数字，而非 numpy 或 list

pybind 直接会类型错误

---

Please throw an exception instead of terminating.